boards/arm/mec15xxevb_assy6853/doc/index.rst

```diff
@@ -0,0 +1,240 @@
+part number MEC1501 144WFBA SOLDER DC ASSY 6860(cpu board) in order to operate.
+
+.. image:: ./mec15xxevb_assy6853.jpg
+     :width: 500px
+     :align: center
+     :alt: MEC15XX EVB ASSY 6853
+
+Hardware
+********
+
+- MEC1501HB0SZ ARM Cortex-M4 Processor
+- 256 KB Ram and 64 KB boot Rom
+- Keyboard interface
+- ADC & GPIO headers
+- UART0, UART1, and UART2
+- FAN0, FAN1, FAN2 headers
+- FAN PWM interface
+- JTAG/SWD, ETM and MCHP Trace ports
+- PECI interface 3.0
+- I2C voltage translator
+- 10 SMBUS headers
+- 4 SGPIO headers
+- VCI interface
+- 5 independent Hardware Driven PS/2 Ports
+- eSPI header
+- 3 Breathing/Blinking LEDs
+- 2 Sockets for SPI NOR chips
+- One reset and VCC_PWRDGD pushbuttons
+- One external PCA9555 I/O port with jumper selectable I2C address.
+- One external LTC2489 delta-sigma ADC with jumper selectable I2C address.
+- Board power jumper selectable from +5V 2.1mm/5.5mm barrel connector or USB Micro A connector.
+
+For more information about the SOC please visit:
+
+- `MEC1501 Reference Manual`_
+
+Supported Features
+==================
+
+The mec15xxevb_assy6853 board configuration supports the following hardware
+features:
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port                         |
++-----------+------------+-------------------------------------+
+
+Other hardware features are not currently supported by Zephyr (at the moment)
+
+The default configuration can be found in the Kconfig file:
+
+        ``boards/arm/mec15xxevb_assy6853/mec15xxevb_assy6853_defconfig``
+
+
+Connections and IOs
+===================
+
+Microchip to provide the schematic for this board.
+
+System Clock
+============
+
+The MEC1501 MCU is configured to use the 48Mhz internal oscillator with the
+on-chip PLL to generate a resulting EC clock rate of 12 MHz. See Processor clock
+control register (chapter 4 in user manual)
+
+Serial Port
+===========
+
+UART0 is configured for serial logs.
+
+Jumper settings
+***************
+
+Please follow the jumper settings below to properly demo this
+board. Advanced users may deviate from this recommendation.
+
+Jump setting for MEC15xx EVB Assy 6797 Rev A1p0
+===============================================
+
+Power-related jumpers.
+
+Power from +5V power brick connected to barrel connector P11 (5.5mm OD, 2.1mm ID)
+JP88 5-6 only
+
+Power from micro-USB type A/B connector P12.
+JP88 7-8 only
+
++-------+------+------+------+------+------+------+------+------+------+------+
```

Comment:
Are these all power-related jumpers too? If not, add another title heading for what these are.